assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -37,6 +37,8 @@ import { Main, Wrapper } from './components';
 import PublisherLogoSettings from './publisherLogo';
 
 const ACTIVE_DIALOG_REMOVE_LOGO = 'REMOVE_LOGO';
+const MIN_IMG_HEIGHT = 96;
+const MIN_IMG_WIDTH = 96;
```

Comment:
Maybe rename so it's clear that these are for publisher logos?

Could be a candidate for `assets/src/dashboard/constants/index.js` too as these might be useful elsewhere in the future.